lte/gateway/c/core/oai/tasks/nas5g/include/ies/M5GQosFlowParam.hpp

```diff
@@ -33,7 +33,7 @@ class M5GQosFlowParam {
 #define M5G_QOS_FLOW_PARAM_BIT_RATE_UNITS_KBPS 1
   uint8_t length;
   uint8_t units;
-  uint16_t element;
```

Comment:
1. add Unittest case for issue and fix.
2. need to fix encode and decode functions according to data type 
3. add UT for encode and decode.

Reply:
Used existing UT for validation.